### PR TITLE
Add / Update French Translation

### PR DESCRIPTION
**Pull request by Elgin675**
**When merged this pull request will:**
- Update / Add French Translation

### CODE REVIEW

File: addons/ace_interact/stringtable.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
             <Japanese>%1 チャンネル: %2</Japanese>
             <Czech>%1 Chn: %2</Czech>
             <Italian>%1 Chn: %2</Italian>
-            <French>%1 Canal: %2</French>
+            <French>%1 Canal : %2</French>
```
Comment (TheMagnetar) [inline addons/ace_interact/stringtable.xml]:
```suggestion
            <French>%1 Canal: %2</French>
```

**Review by TheMagnetar — CHANGES_REQUESTED** [review]

File: addons/ace_interact/stringtable.xml (hunk 1 above)
Comment (Elgin675) [reply addons/ace_interact/stringtable.xml]: Sorry, but I don't understand your Suggested change, @TheMagnetar :) . That doesn't seem correct for me.

Comment (TheMagnetar) [reply addons/ace_interact/stringtable.xml]: Sorry. Typo while being on mobile

Comment (Elgin675) [reply addons/ace_interact/stringtable.xml]: No problems ;).
I'm sorry @TheMagnetar , but the french typographical rules impose a space BEFORE and after a colon.
So I prefer my version ;).

**Comment by alganthe** [comment]
Good translation, I agree with magnetar here, for the sake of consistency (plus it's more redable).

**Comment by Elgin675** [comment]
@TheMagnetar, @alganthe : Your suggestion was committed ;).

**Review by TheMagnetar — APPROVED** [review]

**Comment by TheMagnetar** [comment]
Great and many thanks. @alganthe I would appreciate your review on this again. I am unsure regarding the change from `Placer` to `Installer`.

**Comment by Elgin675** [comment]
@TheMagnetar : Do you have any idea where the _'ACRE_AcreBaseBox'_ and _'ACRE_AllRadioCrate'_ objects are located ? Do they still exist ?

Because I searched everywhere (arsenal, eden, zeus), and I didn't find these objects.
However, they are translated into the _sys_radio\stringtable.xml_ file, lines 115 to 129.

Perhaps they are not required, and should be deleted, but I would need confirmation.

**Review by alganthe — COMMENTED** [review]
Looks good to me

@TheMagnetar i'm not seeing any string being changed from "placer" to "installer",

**Review by jonpas — APPROVED** [review]

**Pull request merged**